_episodes/03-types-conversion.md

```diff
@@ -81,7 +81,7 @@ TypeError: unsupported operand type(s) for -: 'str' and 'str'
 ~~~
 {: .error}
 
-## Strings can be added and multiplied.
+## You can use the "+" and "\*" operators on strings.
```

Comment:
Is it a good idea to include the backslash?  I think this might be confusing to a novice.